src/Mod/Draft/draftguitools/gui_grid.py

```diff
@@ -62,7 +63,7 @@ def GetResources(self):
                                              "Toggles the Draft grid on and off."),
                 "CmdType": "ForEdit"}
 
-    def Activated(self):
+    def Activated(self, index = 0):
```

Comment:
FIXME: I need to find out where this comes from and if it's needed. At the moment, it seems the `index` parameter is unused.